spring-caching/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+#spring.h2.console.enabled=false
```

Comment:
we have this commented out and few lines later it appears again

Reply:
Fixed this